src/network_info.c

```diff
@@ -25,8 +25,8 @@ const network_info_t network_info[NETWORK_INFO_SIZE] =
   { .network_id = NETWORK_ID_LOCAL,
     // 2eNy1mUFdmaxXNj1eQHUe7Np4gju9sJsEtWQ4MX3ToiNKuADed
     .x_blockchain_id = { 0xd8, 0x91, 0xad, 0x56, 0x05, 0x6d, 0x9c, 0x01, 0xf1, 0x8f, 0x43, 0xf5, 0x8b, 0x5c, 0x78, 0x4a, 0xd0, 0x7a, 0x4a, 0x49, 0xcf, 0x3d, 0x1f, 0x11, 0x62, 0x38, 0x04, 0xb5, 0xcb, 0xa2, 0xc6, 0xbf },
-    // 26sSDdFXoKeShAqVfvugUiUQKhMZtHYDLeBqmBfNfcdjziTrZA
-    .c_blockchain_id = { 0x91, 0x06, 0x0e, 0xab, 0xfb, 0x5a, 0x57, 0x17, 0x20, 0x10, 0x9b, 0x58, 0x96, 0xe5, 0xff, 0x00, 0x01, 0x0a, 0x1c, 0xfe, 0x6b, 0x10, 0x3d, 0x58, 0x5e, 0x6e, 0xbf, 0x27, 0xb9, 0x7a, 0x17, 0x35 },
+    // BZycCYv295rhtyykDkJHaGe7foSadXawXT4XTZPqXBrg
+    .c_blockchain_id = { 0x9d, 0x07, 0x75, 0xf4, 0x50, 0x60, 0x4b, 0xd2, 0xfb, 0xc4, 0x9c, 0xe0, 0xc5, 0xc1, 0xc6, 0xdf, 0xeb, 0x2d, 0xc2, 0xac, 0xb8, 0xc9, 0x2c, 0x26, 0xee, 0xae, 0x6e, 0x6d, 0xf4, 0x50, 0x2b, 0x19 },
```

Comment:
Any particular reason this changed?

Reply:
I can ask around for the "why", if it's useful - I changed it for no reason other than: when I ran a local network using the then-latest `avash` and `avalanchego`, these were the 3 blockchain IDs I got back

Reply:
Double-triple checking by asking around internally about the chainID

Reply:
Apparently this should be `2CA6j5zYzasynPsFeNoqWkmTCt3VScMvXUZHbfDJ8k3oGzAPtU`, and not `26sSDdFXoKeShAqVfvugUiUQKhMZtHYDLeBqmBfNfcdjziTrZA` nor `BZycCYv295rhtyykDkJHaGe7foSadXawXT4XTZPqXBrg`. Will fix this and the test cases asap

Reply:
OK!

Reply:
Fixed! I got all turned around: `2CA6j5zYzasynPsFeNoqWkmTCt3VScMvXUZHbfDJ8k3oGzAPtU` should be the correct comment here, but the proposed new bytes `0x9d...0x19` are still correct.